src/index.js

```diff
@@ -66,6 +66,32 @@ const createContext = (ctx) => {
   return ctx
 }
 
+// prefers require, checks for ERR_REQUIRE_ESM, which is still experimental, but often works when called synchronous
+const requireOrImport = filepath => {
+  try {
+    return require(filepath)
+  } catch (error) {
+    // importing ECMAScript modules from CommonJS does not work, so load it using dynamic import(...)
+    if (error.code === 'ERR_REQUIRE_ESM') {
+      return import(filepath).then(mod => mod.default)
+    }
+    throw error
+  }
+}
+
+// prefers import, which won't work in sync mode
```

Comment:
Will it better to check Node.js version?

Reply:
I think feature detection is a bit cleaner, as it might support different runtimes (like Deno). Although in this case, the feature-detection is quite Node-specific and I don't find documentation for it in the Node 10 docs.